server/webdriver_tests/tests/place_explorer_i18n_test.py

```diff
@@ -39,44 +39,45 @@ def test_japan_in_japanese(self):
     # TODO(beets): Re-enable this test after fixing flakiness in finding
     # the chart
     # # Test strings in GDP comparison chart
-    # gdp_chart = self.driver.find_element_by_xpath(
+    # gdp_chart = self.driver.find_element(By.XPATH,
     #     '//*[@id="main-pane"]/section[1]/div/div[5]/div')
     # self.assertEqual(
-    #     gdp_chart.find_element_by_xpath('h4').text, '日本 の 1 人あたりの国内総生産')
+    #     gdp_chart.find_element(By.XPATH, 'h4').text, '日本 の 1 人あたりの国内総生産')
 
     # # Test that chart tick values are translated
-    # y_text = gdp_chart.find_elements_by_class_name(
-    #     'y')[0].find_elements_by_tag_name('text')
+    # y_text = gdp_chart.find_elements(By. CLASS_NAME,
```

Comment:
is the space here intentional?

also the rest of `By. CLASS_NAME` in this file

Reply:
Good catch. This is introduced from batch update.